### PR TITLE
endpoint to return active executions underway on each Orca instance

### PR DESCRIPTION
**Pull request by robfletcher**
relates to  #spinnaker/1394

### CODE REVIEW

File: orca-spring-batch/src/main/java/com/netflix/spinnaker/config/SpringBatchConfiguration.java [hunk 1]
```diff
@@ -33,14 +33,15 @@
 import org.springframework.batch.core.repository.JobRepository;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
 import static java.util.Collections.emptySet;
 
 @Configuration
-@ComponentScan("com.netflix.spinnaker.orca.batch.legacy")
```
Comment (robfletcher) [inline orca-spring-batch/src/main/java/com/netflix/spinnaker/config/SpringBatchConfiguration.java]: This package doesn't even exist so…

File: orca-spring-batch/src/main/java/com/netflix/spinnaker/orca/batch/SpringBatchActiveExecutionTracker.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      .from(jobOperator.getJobNames())
+      .flatMapIterable(this::getRunningExecutions)
+      .count()
+      .subscribe(this::recordCount);
+  }
+
+  private void recordCount(int count) {
+    log.info("Currently running {} executions", count);
+    try (Jedis jedis = jedisPool.getResource()) {
+      jedis.hset(KEY, currentInstance, String.valueOf(count));
```
Comment (robzienert) [inline orca-spring-batch/src/main/java/com/netflix/spinnaker/orca/batch/SpringBatchActiveExecutionTracker.java]: How do we remove instances that no longer exist?

Comment (robfletcher) [inline orca-spring-batch/src/main/java/com/netflix/spinnaker/orca/batch/SpringBatchActiveExecutionTracker.java]: Working on that right now

**Comment by robfletcher** [comment]
@robzienert stale Orcas are expired now

**Comment by robzienert** [comment]
👍 Nice!

**Pull request merged**